### PR TITLE
updated restapi file

### PR DESCRIPTION
**Pull request by sudeep-86**
updated

### CODE REVIEW

File: src/test/java/rahulshettyacademy/Mavenjavanew/RESTAPITest.java [hunk 1]
```diff
@@ -9,6 +9,7 @@ public void postJiraTest() {
 		System.out.println("postJira");
 		System.out.println("postJira2");
 		System.out.println("postJira3");
+		System.out.println("hello updating");
```
Comment (Gurpreet2804) [inline src/test/java/rahulshettyacademy/Mavenjavanew/RESTAPITest.java]: why this change is required

File: src/test/java/rahulshettyacademy/Mavenjavanew/RESTAPITest.java [hunk 2]
```diff
@@ -9,6 +9,7 @@ public void postJiraTest() {
 		System.out.println("postJira");
 		System.out.println("postJira2");
 		System.out.println("postJira3");
```
Comment (Gurpreet2804) [inline src/test/java/rahulshettyacademy/Mavenjavanew/RESTAPITest.java:11]: why this

**Review by Gurpreet2804 — CHANGES_REQUESTED** [review]
pleae check comments

File: src/test/java/rahulshettyacademy/Mavenjavanew/RESTAPITest.java (hunk 2 above)
Comment (sudeep-86) [reply src/test/java/rahulshettyacademy/Mavenjavanew/RESTAPITest.java:11]: hfhffgh

File: src/test/java/rahulshettyacademy/Mavenjavanew/RESTAPITest.java (hunk 1 above)
Comment (sudeep-86) [reply src/test/java/rahulshettyacademy/Mavenjavanew/RESTAPITest.java]: resoved now

**Review by Gurpreet2804 — APPROVED** [review]

**Comment by sudeep-86** [comment]
good to merge
